nrf52840-hal/src/ieee802154.rs

```diff
@@ -26,9 +26,6 @@ pub struct Radio<'c> {
     _clocks: &'c (),
 }
 
-/// Default Clear Channel Assessment method = Carrier sense
-pub const DEFAULT_CCA: Cca = Cca::CarrierSense;
```

Comment:
this should be left as the default CCA mode as it requires no calibration. This is not a particularly good CCA mode but getting the `ed_threshold` wrong results in a worse experience. (I saw the `todo undo` comment below; still pointing out my line of reasoning)